docs/admin/workloads/policies.md

```diff
@@ -51,19 +51,43 @@ The policy places a default and limit on the available values for GPU allocation
 ``` bash
 kubectl apply -f gpupolicy.yaml 
 ```
+
 Now, try the following command:
+
 ``` bash
 runai submit --gpu 5 --interactive -p team-a
 ```
+
 The following message will appear:
+
 ```
 gpu: must be no greater than 4
 ```
-A similar message will appear in the _New Job_ form of the Run:ai user interface, when attempting to enter the number of GPUs, which is out of range for an Interactive tab.  
+
+A similar message will appear in the _New Job_ form of the Run:ai user interface, when attempting to enter the number of GPUs, which is out of range for an Interactive tab.
```

Comment:
Since your example has
kind: TrainingPolicy
need to fix to
```
...er of GPUs, which is out of range for a  training job.
                                           ^^^^^^^^^^^^^^
```

Reply:
```suggestion
A similar message will appear in the _New Job_ form of the Run:ai user interface, when attempting to enter the number of GPUs, which is out of range for an training job.
```
@oferla is this better?